t/lib/WTSI/NPG/HTS/TreePublisherTest.pm

```diff
@@ -279,6 +279,17 @@ sub publish_tree_filter : Test(4) {
       diag explain \@observed_paths;
 }
 
+sub npg_publish_tree_pl_logconf : Test(1) {
+  my $source_path = "${data_path}/treepublisher";
+
+  my @script_args = (
+    q[--collection], ${irods_tmp_coll},
+    q[--source_directory], ${source_path},
+    q[--logconf], q[t/log4perl_test.conf]);
+  ok(system($^X, "${bin_path}/npg_publish_tree.pl", @script_args) == 0,
+    'Correctly exited with logconf option');
```

Comment:
Small issues here:
- The extra braces e.g. `${irods_tmp_coll}` are unnecessary
- In the `system` call, using the built-in `$^X` is unnecesary because you're providing a relative path to a file that has a `#!` header

Reply:
I am realising that I used braces also in the tests done months ago... I will avoid using them from now on when variables are in or out of strings. Thank you